_posts/2020-11-04-ggplot2.md

```diff
@@ -0,0 +1,92 @@
+---
+name: Scott Auerbach
+title:  "Intro to ggplot2 and its parameters"
+date:   2020-11-04
+---
+The goal of this blogpost is to introduce the reader to the powerful tool `ggplot2`
+and provide an explanation and scope of some of the abilities of this tool to create effective
+visualizations of experimental data. By reading this post, the reader will be
+able to use the parameters laid out and write code of their own. Any set of tabular data can be sorted, analyzed and visualized with `ggplot2`.
+
+## General Format
+
+`ggplot2` has three basic components: the data set, which is best provided in
+*.csv* format because it cannot be edited accidentally or intentionally, the
+coordinate system with the Cartesian *x*, *y*, and potentially *z*, and finally the *geom*, which is the visual representation of the data on the graph. Examples of `ggplot2` *geoms* are bar plots, box-and-whisker plots, histograms, and even shapes with the right data set! It is also important to include the function `ggsave` to save the output of the code with your choice of format (*png* is recommended here). The following code chunk shows this:
+
+```{r save-plot-with-ggsave}
+# Save plot here with your filename
+ggsave(plot = nice_plot,
+  filename = "output/figures/nice_plot.png")
+```
+Note that the default for `ggsave` is the last plot you generate, so to prevent
```

Comment:
In the spirit of this, maybe the code chunk you show could also include the actual `ggplot2` call prior to `ggsave` so as to minimize potential confusion?